sites/fast-color-explorer/app/control-pane.tsx

```diff
@@ -360,7 +359,7 @@ class ControlPaneBase extends React.Component<ControlPaneProps, ControlPaneState
         );
     }
 
-    private handleReccomendedBackgroundsChange = (
+    private handleRecommendedBackgroundsChange = (
         e: React.ChangeEvent<HTMLInputElement>
```

Comment:
![image](https://user-images.githubusercontent.com/3213292/81246485-9c80ab80-8fcc-11ea-9ace-223b3addf6ee.png)

Reply:
Seems reasonable. Nothing to be alarmed about here